backend/app/store/engine/bolt.go

```diff
@@ -398,6 +398,15 @@ func (b *BoltDB) ListFlags(req FlagRequest) (res []interface{}, err error) {
 	return nil, errors.Errorf("flag %s not listable", req.Flag)
 }
 
+// ListSites returns slice with all existing site ids
+func (b *BoltDB) ListSites() ([]string, error) {
```

Comment:
Same as with example remote store, this should be covered by a test.

Reply:
I've removed ListSites, because Sites already passed to search at creation stage